src/activitylog/activitylog.html

```diff
@@ -24,41 +24,68 @@
-        <button id="saveLogBtn">Save logs</button>
-        <button id="clearLogBtn">Clear All Logs</button>
-      </div>
-      <div class="notice"></div>
-      <div class="activity-log-wrapper">
-        <h2 class="log-heading">Activity Logs</h2>
+    <div class="content-wrapper">
+      <div class="menu-wrapper">
+        <div class="clear-logs-btn" title="Clear all logs"></div>
+        <div class="separator clear-logs"></div>
         <div class="filter-wrapper">
-          <div class="filter-boxes">
-            <filter-keyword></filter-keyword>
-            <filter-option filter-key="id">
-              Select extensions to filter logs
-            </filter-option>
-            <filter-option
-              filter-key="viewType"
-              title="This filter applies to all log entries except content scripts."
-            >
-              Select view type to filter logs
-            </filter-option>
-            <filter-option filter-key="type">
-              Select api type to filter logs
-            </filter-option>
-            <filter-option
-              filter-key="name"
-              title="This filter only applies to log entries related to API events."
-            >
-              Select api name to filter logs
-            </filter-option>
-          </div>
+          <filter-keyword></filter-keyword>
+          <filter-option filter-key="id">
+            Extension Ids
+          </filter-option>
+          <filter-option
+            filter-key="viewType"
+            title="This filter applies to all log entries except content scripts."
+          >
+            View Types
+          </filter-option>
+          <filter-option filter-key="type">
+            API Types
+          </filter-option>
+          <filter-option
+            filter-key="name"
+            title="This filter only applies to log entries related to API events."
+          >
+            API Names
+          </filter-option>
           <filter-timestamp></filter-timestamp>
         </div>
-        <log-view></log-view>
+        <div class="separator options"></div>
+        <div class="option-menu">
+          <div class="options-btn"></div>
+          <div class="options-dropdown" hidden>
+            <label class="load-file-btn">
+              <input
+                type="file"
+                name="loadLogFile"
+                id="file"
+                class="inputfile"
+                accept="application/JSON"
+              />
+              Load Logs
+            </label>
+            <div class="save-log-btn" id="saveLogBtn">
+              Save All Logs
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="notice"></div>
+      <div class="log-view-wrapper">
+        <log-view />
+      </div>
+      <div class="footer-wrapper">
+        <div class="logs-counter">
+          <span title="Visible Logs">0</span>
+          <div class="separator"></div>
```

Comment:
This should be a slanted separator `/`.

The common UI pattern is `x / y`, meaning `x out of y`.